server/views/case-list.njk

```diff
@@ -104,7 +104,6 @@
 {% set currentDate = moment(params.date, 'YYYY-MM-DD') %}
 {% set today = moment().format('YYYY-MM-DD') %}
 {% set tomorrow = moment().add(1, 'days').format('YYYY-MM-DD') %}
-{% set withinDateRange = currentDate.isBetween(moment(), params.addBusinessDays(moment(), params.totalDays - 1), 'day', '[)') %}
```

Comment:
I'm not sure if we still need this in order to hide the day navigation if the user manually visits a page out of our 7 day window... I've asked Charan for clarification.